### PR TITLE
hiding 'as low as' if all variant prices are same

### PR DESCRIPTION
**Pull request by alex-vo**

### CODE REVIEW

File: src/app/component/ProductActions/ProductActions.component.js
Comment (alfredsgenkins) [inline src/app/component/ProductActions/ProductActions.component.js:414]: Please use destruction of properties.

File: src/app/component/ProductActions/ProductActions.component.js
Comment (alfredsgenkins) [inline src/app/component/ProductActions/ProductActions.component.js:415]: Do not break expressions & operators on to the next line :) There must be a better way!

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]

**Comment by alex-vo** [comment]
Fixed [#1006 ]

**Review by alfredsgenkins — APPROVED** [review]

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]
This only patches behavior on PDP, and does not on PLP. Please implement fixes for PLP as well.

**Comment by alex-vo** [comment]
See also https://github.com/scandipwa/catalog-graphql/pull/69.

**Review by alfredsgenkins — APPROVED** [review]

**Comment by alfredsgenkins** [comment]
This one is approved, BE needs one more check + after BE merge and release the composer version bump will be required.

**Comment by alfredsgenkins** [comment]
Closing, I do not think this calculation must be done on FE.

**Pull request closed without merging**
